run.py

```diff
@@ -1,11 +1,20 @@
 from src.data_ingestion import prepare_data
 from src.plot_map import all_stations_map
 from src.download_data import download_data
+from src.first_analysis import analysis_1
+from src.second_analysis import analysis_2
 
 def main():
+    #Downloading data
     download_data("https://danepubliczne.imgw.pl/data/dane_pomiarowo_obserwacyjne/dane_meteorologiczne/dobowe/opad/")
```

Comment:
Can you add any logic to skip files that are already downloaded?

Reply:
Same for the combined files, if those are present, there is no need to regenerate them every time.